cves/kernel/CVE-2015-6526.yml

```diff
@@ -448,7 +496,11 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
```

Comment:
This is a great description that shows the human element of how vulnerabilities are created.

Reply:
Thank you!